homework5/task1/oop_1.py

```
@@ -0,0 +1,35 @@
+from datetime import datetime, timedelta
+
+
+class Homework:
+    def __init__(self, text: str, deadline: int):
+        self.text = text
+        if deadline >= 0:
+            self.deadline = timedelta(days=deadline)
+        else:
+            print("Deadline must be more then 0, or equal 0")
```

in this case if deadline <0 and you will call self.is_Active() you'll get an error. you should raise an exception here, not print stmt